lib/rubocop/config_validator.rb

```diff
@@ -3,6 +3,7 @@
 module RuboCop
   # Handles validation of configuration, for example cop names, parameter
   # names, and Ruby versions.
+  # rubocop:disable Metrics/ClassLength
```

Comment:
These extra lines push this class length over RuboCop's configured limit.

This class has several multi-line messages, and fairly straight forward logic. I don't really see a way to refactor out parts to make it smaller, without hindering readability. In any case, this isn't something I can take on as part of this contribution.

```
$ bundle exec rubocop lib/rubocop/config_validator.rb
Inspecting 1 file
C
Offenses:
lib/rubocop/config_validator.rb:6:3: C: Metrics/ClassLength: Class has too many lines. [195/192]
  class ConfigValidator ...
  ^^^^^^^^^^^^^^^^^^^^^
1 file inspected, 1 offense detected
```

Reply:
I don't think this is test-only. If you change your test config to this (same cop as you wanted to put in your test):
```yml
Layout/LineLength: Enabled
```

It errors. I guess, it's not really worse than before, probably errors there right now anyways but it seems to suggest to me that this new validation step should happen a bit earlier to catch this. Maybe just move some of the methods around?